docs/assemblies/proc_hotfixing-the-data-plane-rpm-content.adoc

```diff
@@ -0,0 +1,42 @@
+[id="proc_hotfixing-the-data-plane-rpm-content-{context}"]
+= Hotfixing the data plane RPM content
+
+[role="_abstract"]
+
+RPM hotfix content is installed directly on to the data plane nodes.
+
+.Procedure
+
+. Obtain the RPM hotfix content from the source and store it locally.
++
+----
+$ mkdir -p <hotfix_id>/rpms
+$ cp /path/to/hotfix/*.rpm <hotfix_id>/rpms
+----
++
+* Replace `<hotfix_id>` with a hotfix identifier such as a Jira issue, for example `osprh-0000`
+
+. Copy the RPM hotfix content to the affected data plane nodes.
++
+----
+$ ssh <ssh_user>@<data_plane_node> mkdir -p /tmp/<hotfix_id>/rpms
+$ scp <hotfix_id>/rpms/*.rpm <ssh_user>@<data_plane_node>:/tmp/<hotfix_id>/rpms
+----
++
+* Replace `<hotfix_id>` with a hotfix identifier such as a Jira issue, for example `osprh-0000`
+* Replace `<ssh_user>` with the user name used for SSH
+* Replace `<data_plane_node>` with the hostname or IP for the data plane node.
+* Repeat the command for each affected data plane node.
```

Comment:
```suggestion

+
Repeat this step for each data plane node that the hotfix must be applied to.
```
Using a * makes this statement appear in the list of replace statements. I think you want it to stand out on its own as a "Repeat this step". The suggested syntax should make the statement appear at the end of the step. If it skews the output, try taking the line space out from before the +.

I've also suggested changing the statement to read as "repeat the step", as there is more than one command in the instruction.